phonon/fields.py

```diff
@@ -0,0 +1,106 @@
+import itertools
+
+import phonon.connections
+
+
+class Field(object):
```

Comment:
I think it would be good to add PEP-8 style class documentation for the classes in this file. A one-liner that describes what each object is